sceptre/cli/create.py

```diff
@@ -10,9 +10,15 @@
 @click.argument("path")
 @click.argument("change-set-name", required=False)
 @click.option("-y", "--yes", is_flag=True, help="Assume yes to all questions.")
+@click.option(
+    "-dr",
+    "--disable-rollback",
+    is_flag=True,
+    help="Disable the auto rollback and keep resources successfully created or updated",
+)
```

Comment:
Since we're also doing a StackConfig for this option, we'll probably want to provide a CLI override. Rather than using `is_flag`, recommend providing both options:
```suggestion
@click.option(
  "--disable-rollback/--enable-rollback",
  default=None,
  help="Disable the auto rollback and keep resources successfully created or updated",
)
```

See: https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags

Reply:
Great feedback! You're absolutely right, @dboitnot. I didn't even think of that.